modules/ROOT/pages/configuration/server/occ_app_command.adoc

```diff
@@ -190,6 +190,48 @@ Parameters for short form options will be written either directly after the opti
 sudo -u www-data ./occ files:scan -p "user_x/files/folder"  
 ....
 
+[[Brute Force Protection-commands]]
+Brute Force Protection Commands
+-------------------------------
+
+Use these commands to configure the Brute Force Protection app.
+Parametrisation must be done with the `occ config` command set.
+The combination of `uid` and ÌP address` is used to trigger the ban.
+
+List the current settings
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+....
+sudo -u www-data php occ config:list brute_force_protection
+....
+
+Set the setting
+~~~~~~~~~~~~~~~~
+
+To set a new value, use the command below and replace key and value < > accordingly.
+
+....
+sudo -u www-data php occ config:app:set brute_force_protection <Key> --value=< > --update-only
+....
+
+Fail Tolerance [attempts]
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Number of wrong attempts to trigger the ban.
+Default:	3
```

Comment:
This needs to be written as a list or a table for it to render similarly to what the raw text suggests. I suggest a table for the greatest clarity.